lang/python/python/Makefile

```diff
@@ -12,7 +12,7 @@ include ../python-version.mk
 
 PKG_NAME:=python
 PKG_VERSION:=$(PYTHON_VERSION).$(PYTHON_VERSION_MICRO)
-PKG_RELEASE:=7
+PKG_RELEASE:=8
```

Comment:
just a note/heads-up;
there will be a conflict here with PR #9532 ;

Reply:
Right, I'm gonna remove that bump. I will merge this PR once it is also approved from @jefferyto as fast-forward merge and then will apply the #9532.

Reply:
removed bump